pkg/connector/handletwit.go

```diff
@@ -216,3 +217,33 @@ func (tc *TwitterClient) wrapReaction(data *types.MessageReaction, portalKey net
 		TargetMessage: networkid.MessageID(data.MessageID),
 	}
 }
+
+func (tc *TwitterClient) updateTwitterReadReceipt(inbox *response.TwitterInboxData) {
+	for conversationID, conversation := range inbox.Conversations {
+		cache := tc.participantCache[conversationID]
+		for _, participant := range conversation.Participants {
+			if participant.UserID == string(tc.userLogin.ID) {
```

Comment:
as a general note, networkid type conversions should always use dedicated functions. makes it much easier to refactor if the id format were to ever change

```suggestion
			if participant.UserID == ParseUserLoginID(tc.userLogin.ID) {
```